wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioRecorder.swift

```diff
@@ -159,6 +154,27 @@ public final class AudioRecorder: NSObject, AudioRecorderType {
         self.audioRecorder = audioRecorder
     }
 
+    private func makeAudioRecorder(
+        audioFormatID: AudioFormatID,
+        fileURL: URL
+    ) -> AVAudioRecorder? {
+        let settings = [
+            AVFormatIDKey: audioFormatID,
+            AVSampleRateKey: 32000,
+            AVNumberOfChannelsKey: 1
+        ]
+
+        do {
+            let audioRecorder = try AVAudioRecorder(url: fileURL, settings: settings)
+            audioRecorder.isMeteringEnabled = true
+            audioRecorder.delegate = self
+            return audioRecorder
+        } catch {
+            zmLog.error("Failed to initialize `AVAudioRecorder`!")
```

Comment:
nitpic: we could change the log to WireLogger

Reply:
Done with 8bd096b1cd11c2c22ec17a34aecf04b3b21790bb